vipps-login-api.md

```diff
@@ -1101,7 +1117,7 @@ The responses from this endpoint is according to the standard.
 #### Error responses
 In addition to the responses defined by the [standard](https://openid.net/specs/openid-client-initiated-backchannel-authentication-core-1_0.html#rfc.section.11) these responses might be returned:
 
-* `error_code=old_app`: The user's Vipps app is outdated and does not support this login flow.
+* `error_code=outdated_app_version`: The user's Vipps app is outdated and does not support this login flow.
```

Comment:
The error code `old_app` is correct for CIBA flow currently. I guess we should change it to be the same though.